tests/test_behavior/test_video_utils.py

```diff
@@ -203,9 +201,19 @@ def test_custom_chunk_shape(self):
                 [nwbfile.acquisition["imageseries"].data.chunks[i] == j for i, j in enumerate(custom_frame_shape)]
             )
 
-    def test_small_buffer_size(self):
-        frame_size_mb = math.prod(self.frame_shape) / 1e6
-        buffer_size = frame_size_mb / 1e3 / 2
-        video_file = self.create_video(self.fps, self.frame_shape, self.number_of_frames)
-        with self.assertRaises(AssertionError):
-            it = VideoDataChunkIterator(video_file, buffer_gb=buffer_size)
+    # def test_small_buffer_size(self):
+    #     frame_size_mb = math.prod(self.frame_shape) / 1e6
+    #     buffer_size = frame_size_mb / 1e3 / 2
+    #     video_file = self.create_video(self.fps, self.frame_shape, self.number_of_frames)
+    #     with self.assertRaises(AssertionError):
+    #         it = VideoDataChunkIterator(video_file, buffer_gb=buffer_size)
+
+
+# To Discuss in review: This does not raise an error anymore because our implementation has a safety mechanism
```

Comment:
Makes sense. Probably should remove the commented out code above then.

Reply:
Done.